packages/base-map/src/styled.ts

```diff
@@ -1,5 +1,17 @@
+import { Popup as MapGlPopup } from "react-map-gl";
 import styled from "styled-components";
 
+/**
+ * Adds a box shadow and tweaks border radius to make popups easier to read.
+ */
+export const Popup = styled(MapGlPopup)`
+  & > .maplibregl-popup-content,
+  & > .mapboxgl-popup-content {
+    border-radius: 6px;
+    box-shadow: 0 3px 14px rgb(0 0 0 / 40%);
```

Comment:
```suggestion
    box-shadow: 0 3px 14px 4px rgb(0 0 0 / 40%);
```
Could we add some spread to this?

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/45ca046b59d4c46d94a43292084bf17a1e50cec0.

Reply:
Sorry I forgot to add one more change I made which is to change 40% to 20%! which I think looks a bit less jarring.

Reply:
In that case could we revert to the initial shadow which has more contrast and a narrower span?

Reply:
I think with 20% instead of 40% the new one looks much more defined! Just needs to be a little more transparent

Reply:
Ok, see https://github.com/opentripplanner/otp-ui/pull/471/commits/539a6ecb056e69a35cbafe6db03d6024676b30ee.